docs/source/tracking/autolog.rst

```diff
@@ -219,6 +238,56 @@ If early stopping is activated, metrics at the best iteration will be logged as
 
 .. _lightgbm.train: https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.train.html#lightgbm-train
 
+.. _autolog-openai:
+
+OpenAI
+------
+
+Call the generic autolog function :py:func:`mlflow.openai.autolog` before your training code to enable automatic logging of artifacts.
+See an example usage with `OpenAI <https://github.com/mlflow/mlflow/tree/master/examples/openai>`_.
+
+Autologging captures the following information:
+
++-----------+-------------+----------------+---------------+------------------------------------------------------------------------------+
+| Framework | Metrics     | Parameters     | Tags          | Artifacts                                                                    |
++-----------+-------------+----------------+---------------+------------------------------------------------------------------------------+
+| OpenAI    | --          | --             |  --           | - `MLflow Model`_ (OpenAI model) with model signature on training end        |
+|           |             |                |               | - Input example                                                              |
+|           |             |                |               | - Datasets                                                                   |
```

Comment:
@serena-ruan do you know if datasets are logged here? I copy-paste from langchain without a second thought, cuz the code looks quite similar

Reply:
Seems no, and dataset actually doesn't make so much sense for these two flavors, I'll remove them

Reply:
Actually there are a few more flavors that has `log_datasets` in `autolog()` but no-op e.g. `gluon`, `fastai`, `pytorch`, etc. Probably just a copy-and-paste thing, but might be safer to ask team about any historical context.

Reply:
I see. actually datasets are not mentioned in the doc for other flavours (phew). we could make that change in a separate PR